CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## HEAD
 
+* Update dependencies to fix a few CVEs
```

Comment:
It would be great to highlight those important CVEs here.

Reply:
will do that

Reply:
@roger2hk let me know if you want to add anything else

thanks for your review!